turtle_graphics/turtle_graphics_four_quadrants.js

```diff
@@ -0,0 +1,142 @@
+// This was my initial attempt where I tried to solve the problem in a four quadrants x and y axis graph. The program can draw the same thing as the one-quadrant solution does. But the points coordinates are diffrent. It's more like a regular 4-quadrant x-y axis graph, which is different from the one in the assignment's instructions. You can ignore this js file if you do not have enough time. I just want to keep this version for my own reference in the future. Please see the turtle_graphics_one_quadrant.js as my homework solution. Thank you very much!
+
+class Turtle {
+    constructor(x = 0, y = 0) {
+        this.x = x;
+        this.y = y;
+        this.initialX = x;
+        this.initialY = y;
+        this.points = [[x,y]];
+        this.directions = ['east', 'south', 'west', 'north'];
+        this.face = this.directions[0];
+    }
+    
+    forward(step){
+        let oldX = this.x;
+        let oldY = this.y;
+        if (this.face == 'east') {
+            this.x += step; 
+            for (let i = 1; i <= step; i++) {
+                this.points.push([oldX + i, this.y])
+            }
+        }
+        if (this.face == 'west') {
+            this.x -= step;
+            for (let i = 1; i <= step; i++) {
+                this.points.push([oldX - i, this.y])
+            }
+
+        }
+        if (this.face == 'south') {
+            this.y -= step;
+            for (let i = 1; i <= step; i++) {
+                this.points.push([this.x, oldY - i])
+            }
+        }
+        if (this.face == 'north') {
+            this.y += step;
+            for (let i = 1; i <= step; i++) {
+                this.points.push([this.x, oldY + i])
+            }
+        }
+        return this;
+    }
+
+    right(){
```

Comment:
I'm gonna suggest that you can use inheritance next time. For example, you can create a class that `Turtle` can inherit from and it will contain methods such as `right`, `left` and `allPoints` which seem to be the same as one quadrant. This way, code is dry. I'm just being picky lol. Well done!